api/settings.md

```diff
@@ -107,6 +107,17 @@ Status: 200 OK
     "force_update": {
       "enabled": false
     }
+  },
+  "provider": {
+    "providerkey": {
+      "image": "http://provider/image.png",
+      "thumbnail": "http://provider/thumbnail.png",
+      "header": "[?]Please provide the data necessary for booking the parking spaces.",
+      "legal": {
+        "accept": ["http://www.provider.com/terms_and_conditions.html"],
+        "shown": ["http://www.provider.com/terms_and_conditions_info.html"]
+      }
+    }
   }
```

Comment:
Hello,

Please remove me from this thread subscription.

Thanks,
Selim
On May 31, 2016 9:58 AM, "dabio" <notifications@github.com> wrote:

> In api/settings.md
> <https://github.com/parku/docs/pull/54#discussion_r65137123>:
>
> > @@ -107,6 +107,17 @@ Status: 200 OK
> >      "force_update": {
> >        "enabled": false
> >      }
> > +  },
> > +  "provider": {
> > +    "providerkey": {
> > +      "image": "http://provider/image.png",
> > +      "thumbnail": "http://provider/thumbnail.png",
> > +      "header": "[?]Please provide the data necessary for booking the parking spaces.",
> > +      "legal": {
> > +        "accept": ["http://www.provider.com/terms_and_conditions.html"],
> > +        "shown": ["http://www.provider.com/terms_and_conditions_info.html"]
> > +      }
> > +    }
> >    }
>
> I agree about adding ParkU as a provider.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/parku/docs/pull/54/files/7a704bddc73cd90a28608c8678c0753aa940b52a#r65137123>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AEoO4MpH85j6sjoyACExbTtPa7EFVFurks5qG-o8gaJpZM4IiAPw>
> .
>